pipenv/cli.py

```diff
@@ -21,7 +21,7 @@
 
 from .project import Project
 from .utils import (convert_deps_from_pip, convert_deps_to_pip, is_required_version,
-    proper_case, pep423_name, split_vcs)
+                    proper_case, pep423_name, split_vcs)
```

Comment:
This is actually intentional styling that we've used across many of the Kenneth Reitz projects. I think we should probably leave this how it was.